### PR TITLE
Show Total Votes for Delegate

### PR DESCRIPTION
**Pull request by krau612**
Fix for #11 and #1

### CODE REVIEW

File: src/app/pages/address/address.component.ts [hunk 1]
```diff
@@ -29,9 +29,11 @@ export class AddressComponent implements OnInit, OnDestroy {
   public showBalanceFooter: boolean = false;
   public openVoters: boolean = false;
   public votersNumber: number = 4;
+  public supply: number;
```
Comment (luciorubeens) [inline src/app/pages/address/address.component.ts]: Set a default value: `= 0`..

**Comment by luciorubeens** [comment]
Can you take a screenshot how it will look? I can't reproduce here..

**Comment by krau612** [comment]
Here's how the Approval line now looks:

screenshot at 2017-10-28 15-04-32


I just noticed that it always displays in Ark, should I make it display in the selected currency?

**Comment by luciorubeens** [comment]
Thanks! Current token (ARK or DARK), take a look: https://github.com/ArkEcosystem/ark-explorer/blob/master/src/app/components/header/header.component.ts#L7

**Comment by krau612** [comment]
So it should use initCurrency.name instead of currencyName and currencyValue? I think it would make more sense to use the currency name and value that the user has selected because I think every other value changes with the selected currency, such as Forged, Fee, Block Reward, etc.

**Comment by luciorubeens** [comment]
Hmm it makes sense, I think it will seem a bit weird to see the total votes in USD

**Comment by luciorubeens** [comment]
but yeah, to follow the standard you can use the current currency

**Comment by krau612** [comment]
Sounds good :+1:

**Comment by luciorubeens** [comment]
nice! +5 👍🏻

**Pull request merged**